dahu/core/app/scripts/dahuapp.js

```diff
@@ -299,6 +299,7 @@ define('dahuapp', [
     * Save the current project
      */
     function saveScreencast(){
+        events.trigger("app:screencast:save");
```

Comment:
This should be triggered in the controller https://github.com/dahuapp/dahu/blob/next-major-refactor-WIP/dahu/core/app/scripts/controller/screencast.js#L106